DeepCat.js

```diff
@@ -84,6 +77,16 @@
 					$( '#deepcat-hintbox' ).slideUp( 350 );
 				}
 			} );
+
+			$( '#searchInput' ).on( 'keyup', function() {
+				if ( matchesDeepCatKeyword( $( this ).val() ) && !shouldHideHints ) {
+					disableImeAndSuggestions();
+					$( '#deepcat-smallhint' ).slideDown( 100 );
```

Comment:
Please avoid fixed numbers for the milliseconds. When you use `'fast'` the user can customize it in it's user's JS.